core-java-module/src/main/java/programs/String/BinaryString.java

```diff
@@ -0,0 +1,32 @@
+package programs.String;
+/**
+ * Binary String
+ *
+ * @author AnkitRawat
+ *  @since 09 Dec 2022
+ */
+public class BinaryString {
```

Comment:
Binary String is repeated in this PR, either remove it from this PR or decline the other PR.